src/engine/strat_engine/blockdevmgr.rs

```diff
@@ -11,7 +11,7 @@ use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
 use rand::{thread_rng, sample};
-use time::{Duration, Timespec, now};
+use chrono::{Duration, Local, DateTime};
```

Comment:
In theory we alphabetize our use directives, by package and then by name.

So, ```chrono``` directive should go above ```rand```, and the order of names is ```DateTime, Duration, Local```.

I know that ```rand``` directive is not setting the best example, but it's best to leave that alone for this PR.